elasticsearch-agent/src/main/java/com/epam/pipeline/elasticsearchagent/service/impl/BulkRequestSender.java

```diff
@@ -112,4 +140,56 @@ private void indexChunk(final String indexName,
             .collect(Collectors.groupingBy(idConverter::getId))
             .forEach((id, items) -> responsePostProcessor.postProcessResponse(items, objectTypes, id, syncStart));
     }
+
+    private boolean exceedsDocSizeLimit(final DocWriteRequest request) {
+        return RequestChunk.getRequestSizeMb(request) > requestLimitMb;
+    }
+
+    @Getter
+    private static class RequestChunk {
+
+        private double sizeMB;
```

Comment:
Can't we use integer or long for size? Double comparison maybe tricky, and I generally try to avoid it.

Reply:
Comparing size in bytes from now